<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Post_1</class>
 <widget class="QDialog" name="Post_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Post</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>146</red>
           <green>194</green>
           <blue>242</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="HighlightedText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>146</red>
           <green>194</green>
           <blue>242</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="HighlightedText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="HighlightedText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Answer 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Answer 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Answer 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Answer 4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Answer 5</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>146</red>
             <green>194</green>
             <blue>242</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>146</red>
             <green>194</green>
             <blue>242</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>146</red>
             <green>194</green>
             <blue>242</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Answer Question</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_userPic">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_username">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Username</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>View File</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit_questionText">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="plainText">
            <string>Read only</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButton_upPost">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="palette">
              <palette>
               <active>
                <colorrole role="Dark">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>252</red>
                   <green>252</green>
                   <blue>252</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>252</red>
                   <green>252</green>
                   <blue>252</blue>
                  </color>
                 </brush>
                </colorrole>
               </active>
               <inactive>
                <colorrole role="Dark">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>252</red>
                   <green>252</green>
                   <blue>252</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>252</red>
                   <green>252</green>
                   <blue>252</blue>
                  </color>
                 </brush>
                </colorrole>
               </inactive>
               <disabled>
                <colorrole role="WindowText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>252</red>
                   <green>252</green>
                   <blue>252</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Dark">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>252</red>
                   <green>252</green>
                   <blue>252</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Text">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>252</red>
                   <green>252</green>
                   <blue>252</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="ButtonText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>252</red>
                   <green>252</green>
                   <blue>252</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>252</red>
                   <green>252</green>
                   <blue>252</blue>
                  </color>
                 </brush>
                </colorrole>
               </disabled>
              </palette>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Upvotes</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_upPostCount">
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_downPost">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="palette">
              <palette>
               <active>
                <colorrole role="Dark">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>252</red>
                   <green>252</green>
                   <blue>252</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>252</red>
                   <green>252</green>
                   <blue>252</blue>
                  </color>
                 </brush>
                </colorrole>
               </active>
               <inactive>
                <colorrole role="Dark">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>252</red>
                   <green>252</green>
                   <blue>252</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>252</red>
                   <green>252</green>
                   <blue>252</blue>
                  </color>
                 </brush>
                </colorrole>
               </inactive>
               <disabled>
                <colorrole role="WindowText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>252</red>
                   <green>252</green>
                   <blue>252</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Dark">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>252</red>
                   <green>252</green>
                   <blue>252</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Text">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>252</red>
                   <green>252</green>
                   <blue>252</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="ButtonText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>252</red>
                   <green>252</green>
                   <blue>252</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>252</red>
                   <green>252</green>
                   <blue>252</blue>
                  </color>
                 </brush>
                </colorrole>
               </disabled>
              </palette>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Downvotes</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_downPostCount">
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Answers</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
